fendermint/app/options/src/parse.rs

```diff
@@ -94,3 +94,10 @@ pub fn parse_network(s: &str) -> Result<Network, String> {
         }
     }
 }
+
+pub fn parse_eth_address(s: &str) -> Result<Address, String> {
+    match primitives::EthAddress::from_str(&s) {
```

Comment:
What was wrong with the `ethers::core::Address` type?

Reply:
`ethers::core::Address` didn't work directly, and for the provider config we use `f4` addresses, so it was easier to leverage our auxiliary type. But I am not strongly attached to it if you feel there is a simpler way of doing it.